tests/strings/strings.cpp

```diff
@@ -613,6 +618,38 @@ typedef long TestValue_t;
 
 wxGCC_WARNING_SUPPRESS(missing-field-initializers)
 
+static const struct ToIntData
+{
+    const wxChar *str;
+    TestValue_t value;
+    int flags;
+    int base;
+
+    int IValue() const { return value; }
+    unsigned int UIValue() const { return value; }
+
+    bool IsOk() const { return !(flags & Number_Invalid); }
+} intData[] =
+{
+/*    {wxT("1"), 1, Number_Ok},
```

Comment:
Sorry, why was this part commented out? It should be either uncommented (better, I think) or removed entirely.

Reply:
This was for debugging and I forgot to remove the comment. It should be included.